data/model_weight/acpi_AbsPowerModel.json

```diff
@@ -1 +1,16 @@
-{"platform": {"All_Weights": {"Bias_Weight": 220.9079278650894, "Categorical_Variables": {}, "Numerical_Variables": {"bpf_cpu_time_ms": {"scale": 5911.969193263386, "mean": 0, "variance": 0, "weight": 29.028228361462897}}}}}
+{
```

Comment:
These files are vendored from another repo and should be ignored

Reply:
Ok I will restore the json files